dataeng/01-docker-comose/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  minio:
+    image: bitnami/minio:latest
+    restart: always
+    environment:
+      - MINIO_ACCESS_KEY=minio-access-key
+      - MINIO_SECRET_KEY=minio-secret-key
+    volumes:
+      - ./minio:/data
+    ports:
+      - 9000:9000
+
+  db:
+    image: postgres:13.3
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    volumes:
+      - ./postgres-data:/var/lib/postgresql
+    ports:
+      - 5432:5432
+
+  pgadmin:
```

Comment:
maybe we can use this one https://github.com/dbcli/pgcli?

Reply:
@alexeyzavyalov, I couldn't find a Docker image for that in Docker Hub